ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.14.yaml

```diff
@@ -151,9 +151,11 @@ tests:
     - chain: gather-network
     - ref: gather-proxy
     - chain: ipi-vsphere-post
+    - ref: vsphere-deprovision-bastionhost
```

Comment:
There is no need to update release-4.14 configs. Leave that to the bot to sync up.

Reply:
Thanks, I wasn't sure since there are 4.15 configs now. Any idea when the bot will stop forwarding master changes to 4.14?

Reply:
I think whenever fast-forwarding to release-4.14 stops.